CHANGELOG.md

```diff
@@ -6,6 +6,48 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [1.0.6] - 2023-11-15
+
+-    Change name of the package to make it release in ROS-2.
```

Comment:
We can't change the name of a package in a minor version. We can release this package under a different name on ROS 2, but this is different.

Reply:
Sorry I should have remove this line because this PR does not change the package name.